logic/transaction.js

```diff
@@ -526,19 +526,9 @@ class Transaction {
 			}
 		}
 
-		// Check requester public key
+		// Reject if transaction has requester public key
 		if (transaction.requesterPublicKey) {
-			multisignatures.push(transaction.senderPublicKey);
-
-			if (
-				!Array.isArray(sender.multisignatures) ||
-				sender.multisignatures.indexOf(transaction.requesterPublicKey) < 0
-			) {
-				return setImmediate(
-					cb,
-					'Account does not belong to multisignature group'
-				);
-			}
+			return setImmediate(cb, 'Multisig request is not allowed');
```

Comment:
There are many code blocks above this which are dependent on `requesterPublicKey` e.g.

```
if (
 			!transaction.requesterPublicKey &&		 			!transaction.requesterPublicKey &&
 			sender.secondSignature &&		 			sender.secondSignature &&
 			!transaction.signSignature &&		 			!transaction.signSignature &&
 			transaction.blockId !== this.scope.genesisblock.block.id		 			transaction.blockId !== this.scope.genesisblock.block.id
```
We should move this check at top and then update rest of code to clean up `requesterPublicKey` usage.